aliasing/api/character.py

```diff
@@ -44,6 +44,17 @@ def consumables(self):
             self._consumables = [AliasCustomCounter(cc) for cc in self._character.consumables]
         return self._consumables
 
+    def get_consumable(self, name):
+        """
+        Gets the AliasCustomCounter with the name `name`
+
+        :param str name: The name of the custom counter to get.
+        :returns: The custom counter.
+        :rtype: AliasCustomCounter
+        :raises: :exc:`ConsumableException` if the counter does not exist.
+        """
+        return self._get_consumable(name)
```

Comment:
`AliasCharacter._get_comsumable()` returns a `CustomCounter`, not an `AliasCustomCounter` - make sure to wrap that